Src/BinaryConverter/Serializers/ISerializer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BinaryConverter.Serializers
+{
+    public interface ISerializer
+    {
+        bool CommonNullHandle { get; }
```

Comment:
If I get it right, `CommonNullHandle` means `IsNullable` or `CanBeNull`? The current name is not very explanative.